documentation/modules/ROOT/pages/03-demo.adoc

```diff
@@ -7,28 +7,879 @@
+
+To install the Demo application, apply the following manifests.
+
+* https://github.com/cardil/cabs-usvc/blob/main/deploy/apps/legacy.yaml[The legacy application]
+* https://github.com/cardil/cabs-usvc/blob/main/deploy/db/redis.yaml[The Drivers database]
+* https://github.com/cardil/cabs-usvc/blob/main/deploy/apps/drivers.yaml[The Drivers module]
+
+Here are commands to apply the above manifests.
+
+[.console-input]
+[source,shell]
+----
+oc create ns demo
+oc apply \
+  -f cabs-usvc/deploy/db/redis.yaml \
+  -f cabs-usvc/deploy/apps/drivers.yaml \
+  -f cabs-usvc/deploy/apps/legacy.yaml
+
+oc wait ksvc/drivers \
+  --namespace demo \
+  --for condition=Ready=True
+oc wait ksvc/legacy \
+  --namespace demo \
+  --for condition=Ready=True
+----
+
+Here's the expected output
+
+[source,shell]
+----
+namespace/demo created
+pod/redis created
+service/redis created
+service.serving.knative.dev/drivers created
+service.serving.knative.dev/legacy created
+service.serving.knative.dev/drivers condition met
+service.serving.knative.dev/legacy condition met
+----
+
+==== Configuring the Event Mesh
+
+To configure the Event Mesh, apply the following manifests.
+
+* https://github.com/cardil/cabs-usvc/blob/main/deploy/mesh/broker.yaml[_Broker_]
+* https://github.com/cardil/cabs-usvc/blob/main/deploy/mesh/sources.yaml[Sources]
+* https://github.com/cardil/cabs-usvc/blob/main/deploy/mesh/triggers.yaml[Triggers]
+
+Here are commands to apply the above manifests.
+
+[.console-input]
+[source,shell]
+----
+oc apply \
+  -f cabs-usvc/deploy/mesh/broker.yaml \
+  -f cabs-usvc/deploy/mesh/sources.yaml \
+  -f cabs-usvc/deploy/mesh/triggers.yaml
+
+oc wait broker/default \
+  --namespace demo \
+  --for condition=Ready=True
+oc wait sinkbinding/drivers-binding \
+  --namespace demo \
+  --for condition=Ready=True
+oc wait sinkbinding/legacy-binding \
+  --namespace demo \
+  --for condition=Ready=True
+oc wait trigger/trg-drivers \
+  --namespace demo \
+  --for condition=Ready=True
+oc wait trigger/trg-drivers \
+  --namespace demo \
+  --for condition=Ready=True
+----
+
+Here's the expected output
+
+[source,shell]
+----
+broker.eventing.knative.dev/default created
+sinkbinding.sources.knative.dev/drivers-binding created
+sinkbinding.sources.knative.dev/legacy-binding created
+trigger.eventing.knative.dev/trg-drivers created
+trigger.eventing.knative.dev/trg-legacy created
+broker.eventing.knative.dev/default condition met
+sinkbinding.sources.knative.dev/drivers-binding condition met
+sinkbinding.sources.knative.dev/legacy-binding condition met
+trigger.eventing.knative.dev/trg-drivers condition met
+trigger.eventing.knative.dev/trg-drivers condition met
+----
+
+The OpenShift Container Platform provides can provide a clear visualization of our deployed solution.
+
+image::solution-odc.png[width=100%]
+
+The console shows two sink bindings on the left, and they are feeding the events from the applications to the _Broker_ (depicted in the center).
```

Comment:
I'd underline here that the broker is the centralized infrastructure piece that ensures a proper decoupling -